Index: SimpleAnomalyDetector.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\n// Created by avi on 05/11/2021.\n//\n\n#include \"SimpleAnomalyDetector.h\"\n#include <vector>\nvoid SimpleAnomalyDetector ::learnNormal(const TimeSeries &ts) {\n    std::vector<Feature> data = ts.getData2();\n\n    for (int i = 1; i < data.size(); ++i) {\n        int m = 0;\n        int c = -1;\n        for (int j = i; j < data.size(); ++j) {\n            float p = pearson(&data[i].getValues()[0],&data[j].getValues()[0], data[j].getValues().size());\n            if (std::abs(p) > m) {\n                m = p;\n                c = j;\n            }\n            if (c != -1) {\n\n            }\n        }\n    }\n\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/SimpleAnomalyDetector.cpp b/SimpleAnomalyDetector.cpp
--- a/SimpleAnomalyDetector.cpp	(revision 2612949659a2464e87e59625b865d64dc5ba5eb5)
+++ b/SimpleAnomalyDetector.cpp	(date 1636314641790)
@@ -5,8 +5,9 @@
 #include "SimpleAnomalyDetector.h"
 #include <vector>
 void SimpleAnomalyDetector ::learnNormal(const TimeSeries &ts) {
+
+
     std::vector<Feature> data = ts.getData2();
-
     for (int i = 1; i < data.size(); ++i) {
         int m = 0;
         int c = -1;
Index: cmake-build-debug/Testing/Temporary/LastTest.log
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Start testing: Nov 07 20:47 IST\n----------------------------------------------------------\nEnd testing: Nov 07 20:47 IST\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/cmake-build-debug/Testing/Temporary/LastTest.log b/cmake-build-debug/Testing/Temporary/LastTest.log
--- a/cmake-build-debug/Testing/Temporary/LastTest.log	(revision 2612949659a2464e87e59625b865d64dc5ba5eb5)
+++ b/cmake-build-debug/Testing/Temporary/LastTest.log	(date 1636312188137)
@@ -1,3 +1,3 @@
-Start testing: Nov 07 20:47 IST
+Start testing: Nov 07 21:09 IST
 ----------------------------------------------------------
-End testing: Nov 07 20:47 IST
+End testing: Nov 07 21:09 IST
